pkg/properties/propertiesSet_test.go

```diff
@@ -96,14 +96,13 @@ func TestCreatePropertyWithValidNamespaceReturnsOk(t *testing.T) {
 	defer server.Close()
 
 	console := utils.NewMockConsole()
-	expectedOutput := "Successfully created property '" + name + "' in namespace '" + namespace + "'"
 
 	//When
 	err := SetProperty(namespace, name, value, apiServerUrl, console)
 
 	//Then
 	assert.Nil(t, err)
-	assert.Equal(t, expectedOutput, console.ReadText())
+	assert.Equal(t, "", console.ReadText())
```

Comment:
Is this needed? we are expecting nothing back, if there is no error, so as long as there is no error this test should be considered passed